aws/src/test/java/org/apache/iceberg/aws/s3/signer/TestS3RestSigner.java

```diff
@@ -107,7 +107,13 @@ public static void afterClass() throws Exception {
 
     ScheduledThreadPoolExecutor executor =
         ((ScheduledThreadPoolExecutor) validatingSigner.icebergSigner.tokenRefreshExecutor());
-    // token expiration is set to 100s so there should be exactly one token scheduled for refresh
+    // token expiration is set to 10000s so there should be exactly one token scheduled for refresh.
```

Comment:
Could we link or somehow reference the S3SignerServlet class in this comment? That's where it's actually set.

Reply:
sure, I updated the comment to mention that it's set in the servlet